### PR TITLE
Added TBB package to OpenMVS apps, removed previous attempt to do this

### PR DESCRIPTION
**Pull request by scottn-bmg**
Replaced my previous "fix" for OpenMVS CMake builds so that TBB is now added and linked more correctly; each project's CMakeLists.txt file calls find_package on it, and it is added to the project's target_link_libraries. Note that the project still cannot be built in DEBUG due to Assert fails; I think that stuff might be Phil's code and I'm unsure whether I should be touching it. Release works fine.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -205,6 +210,47 @@ else()
...
+	endif()
+	if (TBB_FOUND)
+		include (CGAL_TBB_support)
+		add_compile_definitions(CGAL_LINKED_WITH_TBB)
+	endif()
+	
+	LIST(APPEND OpenMVS_EXTRA_LIBS "CGAL::CGAL;CGAL::TBB_support")
+endif()
+
+set(VCG_ROOT "${CMAKE_SOURCE_DIR}/vcglib")
```
Comment (yurik42) [inline CMakeLists.txt:242]: Looks like VCG_DIR should be defined as CACHE var. Then FIND_PACKAGE(VCG REQUIRED)

File: CMakeLists.txt [hunk 2]
```diff
@@ -205,6 +210,47 @@ else()
...
+	else()
+		MESSAGE("-- Can't find TBB. Continuing without it.'")
+	endif()
+endif()
+
+FIND_PACKAGE(CGAL REQUIRED)
+if(CGAL_FOUND)
+	include(CGAL_TBB_support)
+	if (EXISTS "${CMAKE_SOURCE_DIR}/cgal-5.5.2")
+		set(CGAL_DIR "${CMAKE_SOURCE_DIR}/cgal-5.5.2")
```
Comment (yurik42) [inline CMakeLists.txt:228]: Should be set before FIND_PACKAGE(CGAL REQUIRED) ?

File: libs/MVS/CMakeLists.txt [hunk 3]
```diff
@@ -1,24 +1,4 @@
 
-# Find required packages
```
Comment (yurik42) [inline libs/MVS/CMakeLists.txt:2]: You MUST minimize your changes. Why did you remove that?

File: vcpkg.json [hunk 4]
```diff
@@ -21,7 +21,6 @@
 	"opengl",
     "tbb",
     "tiff",
-    "vcglib",
```
Comment (yurik42) [inline vcpkg.json:24]: This file can be "conditional", the dependency will be included if needed.

File: libs/MVS/CMakeLists.txt (hunk 3 above)
Comment (scottn-bmg) [reply libs/MVS/CMakeLists.txt:2]: I modified this file only to delete things that I had added earlier (undoing my damages...) I had intended to restore it to the state that it was originally in, before branching from Phil's branch, so that my only changes are to the top level CMakeLists.txt. If this works just fine, however, I can leave it

**Review by yurik42 — APPROVED** [review]
